data/data/azure/variables-azure.tf

```diff
@@ -57,6 +57,11 @@ variable "azure_master_root_volume_size" {
   description = "The size of the volume in gigabytes for the root block device of master nodes."
 }
 
+variable "azure_control_ultra_ssd_enabled" {
```

Comment:
Just noticed this:

`control` is inconsistent with `master` in the rest of the variables. I support moving from "master" to "control plane" even if it is inconsistent, which is how I'm reading the intent here. If that's not the intent, I'm fine with "master."

My issue, here, though is I think the variable name may be a little unclear. To me it sounds like this is controlling the ultra ssd enabled, rather than a reference to the control plane. 

Have you considered `azure_control_plane_ultra_ssd_enabled`, `azure_cp_ultra_ssd_enabled` or any other alternatives? Same comment would apply to the gocode variable in tfvars.

Reply:
This is indeed an effort to move toward the compute/controlPlane nomenclature. I recognize the ambiguity of the word "control" but leaned into it because `wc control` == `wc compute`. I am not against using the more verbose `control plane` naming scheme. I'll make those changes now.